zeebe/journal/src/main/java/io/camunda/zeebe/journal/JournalInfoProvider.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.journal;
+
+import java.util.concurrent.CompletableFuture;
+
+public interface JournalInfoProvider {
```

Comment:
:clown_face: Not sure about the name, but I also don't have a better idea :smile: I'm terrible at naming anyway.

Reply:
I agree it's not a good name, hard to find a proper one as you say though.